### PR TITLE
LPS-165257 Add site permission

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -1838,7 +1838,7 @@ private Predicate _getPermissionWherePredicate(
 
 		return _inlineSQLHelper.getPermissionWherePredicate(
 			objectDefinition.getClassName(),
-			dynamicObjectDefinitionTable.getPrimaryKeyColumn());
+			dynamicObjectDefinitionTable.getPrimaryKeyColumn(), groupId);
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1841]: @seltong what happens if the `groupId` is zero (object definition scoped by company)?

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**